keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -1013,6 +1028,12 @@ Session affinity affects a service as a whole for a given consumer. The derived
 service's session affinity will be decided according to the conflict resolution
 policy.
 
+#### Labels and annotations
+
+Labels and Services affects a service as a whole for a given consumer. The derived
+service's labels and annotations will be decided according to the conflict resolution
+if the maps are not identical.
```

Comment:
In the current form I would be in favor of explicitly stating that the maps should be identical (and not the union or things like that) WDYT?

Reply:
Ok - makes sense. Instead of the term "map" which is more of a programming language construct, how about:

"_The derived service's labels and annotations will be decided according to the conflict resolution policy if the set of name/value pairs are not identical between the constituent clusters._"